packages/multichain-network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.1]
+
+### Fixed
+
+- fix: Add missing state change event to `MultichainNetworkController` ([#5331](https://github.com/MetaMask/core.git/pull/5331))
```

Comment:
Nit: We don't usually leave Conventional Commit prefixes in changelog entries, the categories do this for us. We could also be a bit more specific that this relates to the messenger.

``` suggestion
- Add `MultichainNetworkController:stateChange` to list of subscribable `MultichainNetworkController` messenger events ([#5331](https://github.com/MetaMask/core.git/pull/5331))

Reply:
Updated!